ci-operator/step-registry/hypershift/mce/agent/create/agentserviceconfig/hypershift-mce-agent-create-agentserviceconfig-commands.sh

```diff
@@ -33,6 +33,9 @@ function config_agentserviceconfig() {
 apiVersion: agent-install.openshift.io/v1beta1
 kind: AgentServiceConfig
 metadata:
+ annotations:
+  # OCPBUGS-55106 workaround
+  unsupported.agent-install.openshift.io/assisted-service-allow-unrestricted-image-pulls: 'true'
```

Comment:
This would be removed after the bug fix? If that so, could you add a note in the code?